CheapShopWeb/Views/Home/Index.cshtml

```diff
@@ -1,39 +1,44 @@
 ﻿@{
     ViewBag.Title = "Home Page";
 }
+@using (Html.BeginForm(new {controller = "Search", action = "Search"}))
```

Comment:
Ką daro šis line'as?

Reply:
Paspaudus mygtuka yra atidaromas naujas html( kvieciamas search controller ir search action, na o ji returnina view su parametru)

Reply:
Hmm, reikės pasidomėti daugiau apie mygtukų veikimą